SLHCUpgradeSimulations/Geometry/test/ModuleInfo_Phase2.cc

```diff
@@ -299,51 +263,27 @@ void ModuleInfo_Phase2::analyze(const edm::Event& iEvent, const edm::EventSetup&
   std::vector<const GeometricDet*> modules = (*rDD).deepComponents();
   Output << "************************ List of modules with positions ************************" << std::endl;
   // MEC: 2010-04-13: need to find corresponding GeometricDetExtra.
-  std::vector<GeometricDetExtra>::const_iterator gdei(rDDE->begin()), gdeEnd(rDDE->end());
+
   for (unsigned int i = 0; i < modules.size(); i++) {
     unsigned int rawid = modules[i]->geographicalId().rawId();
-    gdei = rDDE->begin();
-    for (; gdei != gdeEnd; ++gdei) {
-      if (gdei->geographicalId() == modules[i]->geographicalId())
-        break;
-    }
-
-    if (gdei == gdeEnd)
-      throw cms::Exception("ModuleInfo") << "THERE IS NO MATCHING DetId in the GeometricDetExtra";  //THIS never happens!
 
     GeometricDet::nav_type detNavType = modules[i]->navType();
     Output << std::fixed << std::setprecision(6);  // set as default 6 decimal digits
     std::bitset<32> binary_rawid(rawid);
     Output << " ******** raw Id = " << rawid << " (" << binary_rawid << ") ";
 
-    //    if ( fromDDD_ && printDDD_ ) {
-    //      Output << "\t nav type = " << detNavType;
-    //    }
-    //nav_type typedef changed in 3_6_2; comment out for now.  idr 10/6/10
-
     Output << std::endl;
     int subdetid = modules[i]->geographicalId().subdetId();
-    double volume = gdei->volume() / 1000;  // mm3->cm3
-    double density = gdei->density() / density_units;
```

Comment:
`density_units` is now unused, please remove it from L90 of this file:
https://github.com/cms-sw/cmssw/pull/31729/files#diff-6b0322842aa80408bfebe8127dd0ad0467828812b3498aafae6251228e33a137R90

This triggers the following errors in compilation: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-abcf69/9934/clang-new-warnings.log